sql-plugin/src/main/spark341db/scala/org/apache/spark/sql/rapids/shims/RapidsErrorUtils.scala

```diff
@@ -21,7 +21,8 @@ package org.apache.spark.sql.rapids.shims
 
 import org.apache.spark.sql.errors.QueryExecutionErrors
 
-object RapidsErrorUtils extends RapidsErrorUtilsBase with RapidsQueryErrorUtils {
+object RapidsErrorUtils extends RapidsErrorUtilsBase
```

Comment:
Likewise here, this shim should include `350db` to support Databricks 14.3.

Reply:
I'm closing this out.  As discussed, we'll chase down Databricks 14.3's changes in a follow-up.